app/controllers/verify/finance_controller.rb

```diff
@@ -31,7 +31,7 @@ def step_name
     end
 
     def confirm_step_needed
-      redirect_to verify_address_path if idv_session.financials_confirmation == true
+      redirect_to redirect_url if idv_session.financials_confirmation == true
```

Comment:
Is it unrelated? Currently the user won't be redirected to a valid route if the feature flag isn't enabled. Are you saying we ignore that and just accept that the code won't work properly with the feature flag disabled, and address it in a separate PR? Or will moving the USPS feature flag check into the routes address this problem?

Sorry if I'm missing the point.